doc/en/example/simple.rst

```diff
@@ -131,21 +131,19 @@ line option to control skipping of ``slow`` marked tests::
         parser.addoption("--runslow", action="store_true",
             help="run slow tests")
 
-    def pytest_runtest_setup(item):
-        if 'slow' in item.keywords and not item.config.getoption("--runslow"):
-            pytest.skip("need --runslow option to run")
-
 We can now write a test module like this::
 
     # content of test_module.py
 
     import pytest
-    slow = pytest.mark.slow
```

Comment:
Thank you for pointing this out @hpk42 :bow:

Shall I change the code snippet accordingly? I'm fine with w/e :smile_cat: